### PR TITLE
Module autogen style updated

### PR DESCRIPTION
**Pull request by Doddox-UV**
Modules generated in separate pages...

### CODE REVIEW

File: docs/source/filter.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Torchsynth.filter
```
Comment (jorshi) [inline docs/source/filter.md:1]: We've removed filters so you can get rid of this file

File: docs/source/index.md [hunk 2]
```diff
@@ -56,9 +56,14 @@ contributing/contributing
 ---
 maxdepth: 3
 name: modules
-caption: Modules
+caption: Torchsynth Packages
 ---
-modules
+defaults
+filter
```
Comment (jorshi) [inline docs/source/index.md:62]: Same here, remove filters

File: requirements.txt [hunk 3]
```diff
@@ -23,7 +23,8 @@ pygments>=2.7.4 # not directly required, pinned by Snyk to avoid a vulnerability
 https://github.com/PyTorchLightning/lightning_sphinx_theme/tarball/master#egg=pt-lightning-sphinx-theme
 sphinxcontrib-napoleon
 mock
-recommonmark
+myst_parser
+linkify-it-py
```
Comment (jorshi) [inline requirements.txt:27]: I'm getting an error that sphinx_rtd_theme package is also required when trying to build. Does that make sense? If so can you add in here?

Comment (jorshi) [reply requirements.txt:27]: And in setup.py too

**Review by jorshi — CHANGES_REQUESTED** [review]
Looks good! A couple comments, afterwards those are addressed this is good to merge.

File: requirements.txt (hunk 3 above)
Comment (Doddox-UV) [reply requirements.txt:27]: Yes it makes sense because "sphinx_rtd_theme" is called by the conf.py file

Comment (Doddox-UV) [reply requirements.txt:27]: So I add it in the configuration: setup.py and requirements.txt

File: docs/source/index.md (hunk 2 above)
Comment (Doddox-UV) [reply docs/source/index.md:62]: So I can remove filter.md from the project. Right?

Comment (jorshi) [reply docs/source/index.md:62]: Yep, go ahead, thank you

Comment (Doddox-UV) [reply docs/source/index.md]: Done

**Pull request merged**